.github/ISSUE_TEMPLATE/content_proposal.md

```diff
@@ -3,16 +3,16 @@ name: Propose a new post/codelab
 about: Use this ticket when you are submitting a new post or codelab for review.
 title: 'content: <Add your title>'
 labels: content proposal
-assignees: ''
-
+assignees: 'kaycebasques'
 ---
 
-**Author**: @[Author's GitHub username.]
-**Reviewer**: [Leave blank. web.dev team will assign a reviewer.]
+**Author**: @[Author's GitHub username]
+**SME Reviewer**: @[SME's GitHub username]
+**web.dev Reviewer**: @[web.dev team will fill this out]
 
 **Target publish date:** <yyyy-mm-dd>
```

Comment:
should we remove this field? I think in practice it tends to be something that folks put stuff into but we never abide by it. Maybe instead we should move the date to the hard deadline checkbox.

```
- [ ] Check this box if you have a hard deadline. Deadline: <yyyy-mm-dd>
```

Reply:
I'm agnostic about whether to remove it. If you do, then I support Rob's proposal. Anything related to a beta release always has a hard deadline.